apps/chat/src/store/application/application.epics.ts

```diff
@@ -83,14 +83,22 @@ const updateApplicationEpic: AppEpic = (action$) =>
           destinationUrl: newApplicationId,
           overwrite: false,
         }).pipe(
-          switchMap(() => of(ApplicationActions.edit(payload.applicationData))),
+          switchMap(() =>
+            concat(
+              of(ApplicationActions.edit(payload.applicationData)),
+              of(ApplicationActions.editSuccess()),
```

Comment:
Shouldn't dispatch `editSuccess` here. Only when updating is completed (line 116)